<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QWidget" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>666</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视唱练耳小助手</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>1201</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="OpenSing">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="OpenListen">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-image: url(:/MyPicture/1.png);
	border:none;
	background-color: rgb(170, 255, 255);
	
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="OpenWrite">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="OpenLog">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>160</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="OpenAIAssistant">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>190</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 12pt &quot;方正小标宋简体&quot;;
     color: #ffffff;
     background-color: #639AD1;
     border-radius: 16px;
     border: none;
}
QPushButton:hover {
    background-color: #7FB3E6;
}
QPushButton:pressed {
    background-color: #4976A5;
}
   
     </string>
   </property>
   <property name="text">
    <string>AI助手 </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
